raiden_contracts/deploy/__main__.py

```diff
@@ -208,6 +208,12 @@ def raiden(
     type=int,
     help="Maximum amount of tokens deposited in UserDeposit",
 )
+@click.option(
+    "--initial_service_deposit",
+    required=True,
+    type=int,
+    help="Maximum amount of tokens deposited in UserDeposit",
```

Comment:
This help text is copy/pasted from above.

Reply:
:man_facepalming: fixed.